core/nodepool/cluster/cluster.go

```diff
@@ -216,6 +210,44 @@ func (c *ClusterRef) validateWorkerRootVolume(ec2Svc ec2CreateVolumeService) err
 	return nil
 }
 
+func (c *ClusterRef) getWorkerRootVolumeConfig() *ec2.CreateVolumeInput {
+	var workerRootVolume = &ec2.CreateVolumeInput{}
+
+	switch c.RootVolume.Type {
+	case "standard":
```

Comment:
Almost nit but `case "standard", "gp2":` would be a bit more concise!

Reply:
Certainly